src/makeCalculator.js

```diff
@@ -4,7 +4,38 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    get result() {
```

Comment:
The getter for `result` is unnecessary. The `result` should be a direct property of the returned object, not accessed via a getter.